contrib/action_recognition/README.md

```diff
@@ -1,28 +1,41 @@
 # Action Recognition
 
-This is a place holder. Content will follow soon.
+This folder contains projects relevant to video-based action recognition.
```

Comment:
Would be great if you could a bit of an introduction, and also highlight that we recommend to use r2+1d and that in fact e.g. the live demo is only implemented for r2+1d. i3d is only provided for reference, in case somebody wants to try out another approach. 
See also for introduction (would be great to follow a similar layout even!!):
https://github.com/microsoft/ComputerVision/tree/master/scenarios/classification

Reply:
Added. We may put more effort in explaining basics of action recognition once we merge this into the core.